templates/app/controllers/spree/checkout_controller.rb.tt

```diff
@@ -16,14 +16,20 @@ class CheckoutController < Spree::StoreController
     before_action :ensure_valid_state
 
     before_action :associate_user
+
+    <%- if include_authentication? -%>
```

Comment:
`include_authentication?` is a private method of SolidusStarterFrontendGenerator. When the Thor templates are processed by the SolidusStarterFrontendGenerator, the object that receives the messages from the ERB code is the generator instance.